Core/Relationships/RelationshipResolver.cs

```diff
@@ -396,10 +457,10 @@ public List<CkanModule> ModList()
 
         public bool IsConsistant
         {
-            get { return conflicts.Any(); }
+            get { return !conflicts.Any(); }
```

Comment:
It is used in a single test which this PR enables. This PR is pre repo merge so the GUI wasn't changed to make use of the new helper yet.